sections/main-cart-items.liquid

```diff
@@ -105,6 +105,28 @@
                         </li>
                       {%- endfor -%}
                     </ul>
+                    {%- if item.original_price != item.final_price -%}
+                      <dl class="cart-item__discounted-prices">
+                        <dt class="visually-hidden">
+                          {{ 'products.product.price.regular_price' | t }}
+                        </dt>
+                        <dd>
+                          <s class="cart-item__old-price product-option">
```

Comment:
I used the existing class `product-option` to ensure the styles are the same. This is not technically an option so maybe we can rename it or create a new class with the same styles? Any thoughts?